GSA/digitalgov.gov
# Challenges and prizes topic updates

## Summary

Updating "Challenges and prizes" topic to match new Topic page format.

https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/ss-challenges-and-prizes-topic-updates/topics/challenges-and-prize-competitions/ 

### Preview

[Link to Preview]()



### Solution
Added new content to challenges and prizes page and updated page to match new Topics page format.

### How To Test

1. Navigate to "Challenges & Prize Competitions" topic on Digital.gov.
2. Ensure that content matches what was drafted for this page and approved by Toni and Ammie
3. Ensure that communities and resources referenced include the correct kicker text

---

### Dev Checklist
